WordPress/wordpress-playground
# Support mutliple sites when using browser storage

## What is this PR doing?

It adds support for storing multiple sites in browser storage.

Up until this PR only one site could be stored in a browser. If you wanted to add another site, you would need to overwrite the first one. With this PR you are able to add more sites without affecting existing sites. 

## What problem is it solving?

There is no way to run multiple Playground sites with browser storage enabled because sites override each other.

## How is the problem addressed?

By adding a `site-slug` query argument that allows users to specify which site to load.

## Testing Instructions

- Checkout this PR
- Open this URL http://localhost:5400/website-server/?storage=browser
- Make changes to the site
- Open this URL http://localhost:5400/website-server/?storage=browser&site-slug=test
- Confirm that it's a new site
- Make changes
- Refresh and confirm the changes are still there
- Go back to the first URL and confirm it still has the original changes
